sys-boot/grub/files/grub-2.06-add-gpt-partition-scheme.patch

```diff
@@ -0,0 +1,2783 @@
+From 026cab86883e256607f6df30375bfda4468ae71e Mon Sep 17 00:00:00 2001
+From: Michael Marineau <michael.marineau@coreos.com>
+Date: Sun, 28 Sep 2014 21:26:21 -0700
+Subject: [PATCH] gpt: start new GPT module
```

Comment:
There is also
```
commit e49d5b587e46cd4611be2a5a18d2c2f8c9a40281
Author: Michael Marineau <michael.marineau@coreos.com>
Date:   Mon Nov 3 17:14:37 2014 -0800

    gpt: new gptprio.next command for selecting priority based partitions
```
and I don't see it as separate patch, yet it seems the contents of it are here - did you merge both patches or is this a newer patch from a different repo or branch and not found as is on the flatcar-master branch?

Reply:
In any case it  would be good to add all the details in the commit message of your "sys-boot/grub: Apply Flatcar patches" commit.